libs/application/template-api-modules/src/lib/modules/templates/accident-notification/accident-notification.service.ts

```diff
@@ -78,6 +82,14 @@ export class AccidentNotificationService {
       }
     } catch (e) {
       this.logger.error('Error submitting application to SÍ', e)
+      // In the case we get a precondition error we present it to the user
+      if (e.body && e.body.errorList && e.body.errorList.length > 0) {
+        throw new Error(
+          `Villa kom upp við vistun á umsókn. ${e.body.errorList
+            .map((e: any) => e.errorDesc)
```

Comment:
Is this safe or even informative in all cases to propogate to the client? 
I would think an error of "Innri villa í kerfum Sí" or "Villa við staðfestingu á heilleika skjals. Villuboð: ORA-30937... " would cause major confusion or atleast some kind of panic.

Reply:
Good point, ill get a list of possible errors we can get from the precondition failed and whitelist the ones we want to present to the users. Putting the task on hold until then.